workshops/book-dash/2023-Nov-BookDash-10.md

```diff
@@ -0,0 +1,318 @@
+# Book Dash, November 2023 Report
+
+
+## The Turing Way Book Dash, 13-17 November 2023: Overview and Report
+
+*[Read more about The Turing Way Book Dash here](https://the-turing-way.netlify.app/community-handbook/bookdash.html)*
+
+We are delighted to share that the tenth Book Dash was hosted successfully from 13 to 17 November 2023 with 32 participants.
+
+
+
+![Alt: Screenshot of 18 zoom windows of people smiling. Alex has a border of emojis around her head. The Ada Zoom Room shows 6 people seated in front of their screens along a long conference room. Anne has a zoom background from the meme "THIS IS FINE" which is a cartoon rendering of flames in a home, with smoke filling the top of the screen. She is wearing a tan jacket. Alejandro is wearing overear black headphones. He is holding Matilda beneath him, a small baby with brown hair. Batool is wearing a maroon scarf, and has her hands up. She has a filter that gives her a seedling hat and red blush on her cheeks. Patricia has a green shirt on, and is smiling with black headphones. Maria has a filter that adds christmas lights above her. She has curly hair and a grey t-shirt. Liz is wearing a red shirt, and has a white wall and bookshelf behind her. David has both hands up, and is wearing a green shirt. Alex has a ping shirt and has both hands up, while smiling. Kirstie has a grey hoodie and black glasses. She is sitting next to Mackenzie, who is wearing a green shirt. Jesica has black overear headphones and is smiling with a window next to her. Arielle has a filter that is giving her a pirate hat and eye patch. Her background is a Scriberia image of The Turing Way chapters. Anne (F) is smiling while wearing a white scarf and light blue top.](https://hackmd.io/_uploads/S1xWFkRST.jpg)
+
+
+_Screenshot of the second Community Share-outs, hosted on 17 November 2023. From top to bottom, left to right: Alexandra Araujo Alvarez, Anne Lee Steele, Jim Madge, Cass Goud van Praag, Ada Zoom Room, Kirstie Whitaker, Sarah Gibson, David Llewellyn-Jones, Jennifer Ding, Fran Gomez, David Sarmiento, Arielle Bennett, Batool Almarzouq, Susana Roman Garcia, Sophia Batchelor, Ceilidh Welsh, Saranjeet Kaur (not pictured)._
+
+Accepted participants, trainers and presenters in alphabetical order:
+
+Anne Lee Steele (*), Alejandro Coca-Castro, Alexandra Araujo Alvarez (*), Arielle Bennett (*), Batool Almarzouq (*), Calum Heath (Scriberia illustrator), Cari Hyde-Vaamonde, Ceilidh Welsh, Emma Karoune (*), Eriol Fox, Esther Plomp (*), Francisco Gomez-Medina, David Llewellyn-Jones, David Sarmiento, Gigi Kenneth, Jennifer Ding, Jesica Formoso, Johanna Bayer, Jonny Heath (Jonny Writes words - workshop facilitator), Kalle Westerling, Kirstie Whitaker, Liz Hare (*), Malvika Sharan, Maria Cristina Nanton, Mike Nolan, Patricia Andrea Loto, Richard J. Acton, Sarah Gibson, Sara Villa, Saranjeet Kaur (*), Susana Roman-Garcia (*) and Winny Nekesa (*)
```

Comment:
Anne Lee Steele (*), Alejandro Coca-Castro, Alexandra Araujo Alvarez (*), Arielle Bennett (*), Batool Almarzouq (*), Calum Heath (Scriberia illustrator), Cari Hyde-Vaamonde, Ceilidh Welsh, Emma Karoune (*), Eriol Fox, Esther Plomp (*), Francisco Gomez-Medina, David Llewellyn-Jones, David Sarmiento, Gigi Kenneth, Jennifer Ding, Jesica Formoso, Johanna Bayer (*), Jonny Heath (Jonny Writes words - workshop facilitator), Kalle Westerling, Kirstie Whitaker, Liz Hare (*), Malvika Sharan, Maria Cristina Nanton, Mike Nolan, Patricia Andrea Loto, Richard J. Acton, Sarah Gibson, Sara Villa, Saranjeet Kaur (*), Susana Roman-Garcia (*) and Winny Nekesa (*)